libs/core/src/lib/slider/slider.component.html

```diff
@@ -0,0 +1,162 @@
+    <div class="fd-slider__track" (click)="onTrackClick($event)" #track>
+        <ng-container *ngIf="!_isRange">
+            <div *ngIf="!hideProgressBar" class="fd-slider__track-range" [style.width.%]="_progress"></div>
+            <div
+                #handle
+                class="fd-slider__handle"
+                [attr.tabindex]="disabled ? -1 : 0"
+                role="slider"
+                aria-label="slider"
+                [fdPopover]="singleSliderPopover"
+                [triggers]="[]"
+                [closeOnOutsideClick]="false"
+                [appendTo]="handle"
+                [disabled]="!tooltipMode"
+                placement="top"
+                [attr.aria-valuemin]="min"
+                [attr.aria-valuemax]="max"
+                [attr.aria-valuenow]="getValuenow(_position, _sliderValueTargets.SINGLE_SLIDER)"
+                [style.left.%]="!_isRtl ? _progress : null"
+                [style.right.%]="_isRtl ? _progress : null"
+                (mousedown)="onHandleClick($event); $event.stopPropagation()"
+                (keydown)="onKeyDown($event)"
+            ></div>
+        </ng-container>
+
+        <ng-container *ngIf="_isRange">
+            <div
+                *ngIf="!hideProgressBar"
+                class="fd-slider__track-range"
+                [style.width.%]="_rangeProgress"
+                [style.left.%]="
+                    !_isRtl ? (_handle1Position < _handle2Position ? _handle1Position : _handle2Position) : null
+                "
+                [style.right.%]="
+                    _isRtl ? (_handle1Position < _handle2Position ? _handle1Position : _handle2Position) : null
+                "
+            ></div>
+
+            <div
+                #rangeHandle1
+                class="fd-slider__handle"
+                [attr.tabindex]="disabled ? -1 : 0"
+                role="slider"
+                aria-label="slider"
+                [fdPopover]="rangeSliderPopover1"
+                [triggers]="[]"
+                [closeOnOutsideClick]="false"
+                [appendTo]="rangeHandle1"
+                [disabled]="!tooltipMode"
+                placement="top"
+                [attr.aria-valuemin]="min"
+                [attr.aria-valuemax]="max"
+                [attr.aria-valuenow]="getValuenow(_handle1Value, _sliderValueTargets.RANGE_SLIDER1)"
+                [style.left.%]="!_isRtl ? _handle1Position : null"
+                [style.right.%]="_isRtl ? _handle1Position : null"
+                (mousedown)="onHandleClick($event); $event.stopPropagation()"
+                (keydown)="onKeyDown($event)"
+            ></div>
+
+            <div
+                #rangeHandle2
+                class="fd-slider__handle"
+                [attr.tabindex]="disabled ? -1 : 0"
+                role="slider"
+                aria-label="slider"
+                [fdPopover]="rangeSliderPopover2"
```

Comment:
Could you change it to `[fdPopoverTrigger]="[fdPopoverTrigger]="popover""` 
and instead of `<ng-template #rangeSliderPopover` change it to  `<fd-popover #rangeSliderPopover2`?.
This will require moving all options (triggers/closeOnOutisdeClick/etc) to the `fd-popover #...`
PopoverDirective has been deprecated and shouldn't be used anymore